desktop/renderer-app/src/AppRoutes/AppRouteErrorBoundary.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ComponentType } from "react";
+import { RouteComponentProps } from "react-router-dom";
+import { ErrorPage } from "flat-components";
+
+export interface AppRouteErrorBoundaryProps {
+    Comp: ComponentType<any>;
+    title: string;
+    routeProps: RouteComponentProps;
+}
+
+export interface AppRouteErrorBoundaryState {
+    hasError: boolean;
+}
+
+export class AppRouteErrorBoundary extends React.PureComponent<
```

Comment:
Can it be rewritten as a function component

Reply:
@BlackHole1 https://stackoverflow.com/questions/48482619/how-can-i-make-use-of-error-boundaries-in-functional-react-components